src/Illuminate/Foundation/Auth/ResetsPasswords.php

```diff
@@ -100,10 +100,9 @@ protected function credentials(Request $request)
      */
     protected function resetPassword($user, $password)
     {
-        $user->forceFill([
-            'password' => bcrypt($password),
-            'remember_token' => Str::random(60),
-        ])->save();
+        $user->forceFill(['password' => bcrypt($password)]);
```

Comment:
In this case we can avoid to call `forceFill`; `$user->password = bcrypt($password);` is just fine